File: frontend/desktop/lib/models/requests/set_password_request.dart
/// Body for POST /admins/{id}/set-password — SuperAdmin directly sets a
/// staff/organization account's password (no current password needed,
/// unlike self-service ChangePasswordRequest).
class SetPasswordRequest {
  final String newPassword;
  final String confirmPassword;

  const SetPasswordRequest({required this.newPassword, required this.confirmPassword});

  Map<String, dynamic> toJson() => {
        'NewPassword': newPassword,
        'ConfirmPassword': confirmPassword,
      };
}

File: backend/services/identity/eTicketing.Identity.Business/Admins/Dtos/SetPasswordRequest.cs
namespace eTicketing.Identity.Business.Admins;

/// <summary>SuperAdmin directly sets a staff/organization account's password (no current
/// password needed — see AdminService.SetPasswordAsync). Deliberately excludes User and
/// SuperAdmin targets.</summary>
public sealed record SetPasswordRequest
{
    public string NewPassword { get; init; } = string.Empty;
    public string ConfirmPassword { get; init; } = string.Empty;
}
